### PR TITLE
Use public-master when starting openshift

### PR DESCRIPTION
**Pull request by danmcp**

### CODE REVIEW

**Comment by danmcp** [comment]
@derekwaynecarr @jwforres Review please

File: lib/vagrant-openshift/action/install_openshift3.rb [hunk 1]
```diff
@@ -29,6 +29,14 @@ def call(env)
           sudo(env[:machine], %{
 set -x
 
+#systemctl enable firewalld
```
Comment (jwforres) [inline lib/vagrant-openshift/action/install_openshift3.rb]: assuming these are commented out for a reason?

Comment (danmcp) [inline lib/vagrant-openshift/action/install_openshift3.rb]: @jwforres firewalld is currently disabled.  I need to track down why.  I was just adding these for when we do enable it on these vms.

**Comment by derekwaynecarr** [comment]
Should I be able to try this out on a new AMI if I rebuild from this branch?

**Comment by danmcp** [comment]
@derekwaynecarr You can start with with the deps image and run the steps in openshift3_ami.  That's what I was doing yesterday.

**Comment by danmcp** [comment]
@derekwaynecarr @jwforres I just realized this is totally in the wrong place and needs to be done on provision.

**Pull request merged**